rider/frontend.gradle

```diff
@@ -185,7 +185,7 @@ prepareSandbox {
     dependsOn prepareForBuild
 
     doLast {
-        assert file(unityeditorplugin_path).exists()
+        //assert file(unityeditorplugin_path).exists()
```

Comment:
is this intended?

Reply:
I need your advice on this, lets talk today.